### PR TITLE
DSK-18281 Update the add-in developer guide and examples for the newly added "Selection" API

### PR DESCRIPTION
**Pull request by zhanglin-wu**
Here are the updates:
1. Add description into the developer guide for the newly added API "Selection"
2. Add an add-in example to demonstrate the usage of the newly added API
3. Update the snapshots of the existing add-in examples so that they are consistent with the newly added add-in example which is using Windows 10 style dialogs

### CODE REVIEW

**Comment by zhanglin-wu** [comment]
@PerkinElmer/chemdraw-desktop-reviewers, @ASmellie, Please review.

Note: The add-in guide (PDF file) is already reviewed by Andrew and the team.

File: Examples/Selection Monitor/chemdraw-addin-metadata.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "description": "This is an example of ChemDraw add-in that demostrates the usage of the selection API.",
```
Comment (abhimanyusirohi) [inline Examples/Selection Monitor/chemdraw-addin-metadata.json]: type: `demostrates`  ==> `demonstrates`

Comment (abhimanyusirohi) [reply Examples/Selection Monitor/chemdraw-addin-metadata.json]: I think it should be `This is an example ChemDraw add-in that demonstrates the usage of the selection API.`

File: Examples/Selection Monitor/selection-monitor.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+//
+// delection-monitor.js
```
Comment (abhimanyusirohi) [inline Examples/Selection Monitor/selection-monitor.js]: selection-monitor?

File: Examples/README.md [hunk 3]
```diff
@@ -12,4 +12,10 @@ A simple ChemDraw add-in that shows the version of the ChemDraw JavaScript API.
...
 
-![Document Data Importer and Exporter](./Images/document-data-importer-and-exporter.png)
\ No newline at end of file
+![Document Data Importer and Exporter](./Images/document-data-importer-and-exporter.png)
+
+## Selection Monitor
+
+A ChemDraw add-in that uses the API to get the preview image of the selected structures in the active document.
```
Comment (abhimanyusirohi) [inline Examples/README.md]: `that uses the selection API to get...`?

**Comment by zhanglin-wu** [comment]
All issues are addressed. Please review.

**Comment by abhimanyusirohi** [comment]
@zhanglin-wu This is open for a very long time

**Comment by zhanglin-wu** [comment]
We need the approvals for this PR, but we should hold on the merge until the day when ChemDraw 18 is officially released.

**Pull request merged**